src/components/Header/HeaderWrapper/HeaderWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import useNavbar from "../../../hooks/useNavbar";
+import ActiveHeaderSubItem from "../HeaderWrapperSubItem/ActiveSubItem/ActiveHeaderSubItem";
+import UnactiveHeaderSubitem from "../HeaderWrapperSubItem/UnactiveSubItem/UnactiveHeaderSubItem";
+import styles from "./HeaderWrapper.module.css";
+import ReactDOM from "react-dom";
+const HeaderWrapper = () => {
+  const [activeTab, setActiveTab] = useState<number | null>(null);
+  const handleActiveTab = (index: number) => {
+    if (activeTab != null && activeTab === index) {
+      setActiveTab(null);
+    } else {
+      setActiveTab(index);
+    }
+  };
+  const items = useNavbar();
+  return ReactDOM.createPortal(
+    <div className={styles.navbar_wrapper}>
+      <div className={styles.navbar_subitems}>
+        {items &&
+          items.length > 0 &&
+          items.map((el, index) =>
+            index === activeTab ? (
+              <ActiveHeaderSubItem
```

Comment:
Why we can't create one component and pass prop - active to him, as i see a lot of duplicated code